ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.6-community__okd.yaml

```diff
@@ -0,0 +1,46 @@
+binary_build_commands: make build
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: build/Dockerfile.ci
+  to: windows-machine-config-operator-test
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 600Mi
+tag_specification:
+  name: "4.6"
+  namespace: origin
+tests:
+- as: lint
+  commands: make lint
+  container:
+    from: src
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: unit
+  commands: make unit
+  container:
+    from: src
+- as: aws-e2e-operator
+  steps:
+    cluster_profile: aws
+    test:
+    - ref: windows-e2e-operator-test
+    workflow: ipi-aws-ovn-hybrid
+- as: aws-e2e-operator-with-custom-vxlan-port
```

Comment:
I suggest leaving this out. This will eventually be targeted at vSphere. I don't see folks on the community side using a custom VXLAN port with AWS.